tests/UKHO.ERPFacade.API.FunctionalTests/Model/LUpdatedJsonPayloadHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Newtonsoft.Json;
```

Comment:
Need to remove this file as currently it is not used

Reply:
Done